curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65fa9402d1fb5516aa42159d.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 65fa9402d1fb5516aa42159d
+title: Step 46
+challengeType: 20
+dashedName: step-46
+---
+
+# --description--
+
+Feel free to modify `v3` so that it sums an `R2Vector` instance with an object of different type, such as an integer or a string. You will be able to see the custom error message printed to the console.
```

Comment:
```suggestion
Feel free to modify `v3` so that it sums an `R2Vector` instance with an object of a different type, such as an integer or a string. You will be able to see the custom error message printed on the console.
```